plotting/tile.py

```diff
@@ -140,6 +140,9 @@ def scale(args):
     formatter.set_powerlimits((-3, 4))
     bar = ColorbarBase(ax, cmap=cmap, norm=norm, orientation='vertical',
                        format=formatter)
+    if variable_name == 'Potential Sub Surface Channel':
+        bar.set_ticks([0,1],True)
+    
```

Comment:
Bit hacky but I'll let it pass :joy:

Reply:
Yeah, I know. Could not find a better place to do this :smile: